### ISSUE TITLE
Custom Repos?

### ISSUE THREAD

**Issue by Jan69** [Fox2Code/FoxMagiskModuleManager#131]
**Is your feature request related to a problem? Please describe.**
Module repos are preset and fixed, with no way to edit.

**Describe the solution you'd like**
Some way to edit the Repo list, preferably in-app.

**Describe alternatives you've considered**
Manually recompiling the app every time there's an update or I want to add a repo would be way too annoying and time-consuming, as would manually sending a bunch of modules to a bunch of devices.

**Additional context**
I would like to have ability to use custom repos, which might not be good enough quality for publically listing, but would be nice to have personally, e.g. personal test modules, which might be buggy, but would be convenient to not have to manually transfer, possibly to several devices, or the old deprecated repo's contents (some of the modules haven't been mirrored to the fork repo), etc
Basically stuff that's not high-quality enough, at our own risk (just like f-droid allows custom repos and android allows third-party app installs, both being your own responsibility to make sure are safe)

**Comment by Fox2Code** [comment]
Actually I already added string translation for add/remove repo in the app, adding and removing repo is a planed feature.
The code is also made to support adding repos easily.

Just no one made an issue about it yet, so I use this issue to keep track of it.

Also yeah sadly I can't make equal publicity to all repos, and I had to make compromises.
Old repo have a lot of broken links, so I had to remove it from the app.

**Comment by Fox2Code** [comment]
This should work now on  [`0.6.0-dev1`](https://github.com/Fox2Code/FoxMagiskModuleManager/releases/tag/0.6.0-dev1).
